### DISCUSSION TITLE
Add an option to hide the fact that you got new notifications

### DISCUSSION THREAD

**Discussion by janneslessmann** [ppy/osu#16386] [Ideas]
Sometimes you can get distracted by the red icon on the top right of the screen.

![osu-notification](https://user-images.githubusercontent.com/88435053/148688196-76f9cb01-7aa1-442e-916c-225c90f28e88.png)

So maybe there could be a feature to hide this red icon.

**Comment by Theighlin** [comment]
When would you get distracted, outside of gameplay? You are supposed to dismiss them if you've aknowledged what the notifications are about. Isn't that how it works in most places? I think at best we could get a "notifications" section and be able to toggle stuff. I get annoyed when i import multiple maps and would like it to not show (just tell me if it failed?), but i still wanna know when people message me and others.

**Reply by janneslessmann** [reply]
That sounds exactly like the stuff I had in mind

**Reply by Theighlin** [reply]
although i know that here they don't like toggles too much... (and for good reasons)

**Reply by plaguemask** [reply]
> I get annoyed when i import multiple maps and would like it to not show

Definitely agree. Why would I need to be notified that the game did what I told it to do... in a way that commands attention and needs to be actively dismissed... even after taking actions that make the information certainly unnecessary, like navigating to or playing the map that was imported? It feels like clutter.
At the very least, the counter shouldn't be red for them. Notifications of success don't need the attention.

**Comment by bdach** [comment]
probably could hook into the ["do not disturb"](https://github.com/ppy/osu/blob/f5a5b87cf6e633e237b7595b1ef9cbf3dffe3da7/osu.Game/Overlays/Login/UserAction.cs#L12-L13) user status that is currently there just for display.
